src/main/scala/fs2/aws/s3/s3.scala

```diff
@@ -34,7 +34,7 @@ package object s3 {
           case Left(_) => fs2.Pull.eval(F.delay(None))
         }
         .flatMap {
-          case Some(o) => fs2.Pull.outputChunk(o) >> go(offset + o.size)
+          case Some(o) => fs2.Pull.output(o) >> go(offset + o.size)
```

Comment:
was it something that changed with the API update or was it wrong in the first place?

Reply:
https://github.com/functional-streams-for-scala/fs2/blob/v1.0.0/docs/migration-guide-1.0.md#chunks-and-segments

Changed with the API update